lib/passpartu/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Passpartu
-  VERSION = '0.6.0'.freeze
+  VERSION = '0.6.0'
```

Comment:
Upgrade version to 0.7.0
Add changes to README as well as a description of new files configuration process!